app/src/main/java/com/thenewboston/kotlinsdk/home/viewmodels/BankViewModel.kt

```diff
@@ -0,0 +1,13 @@
+package com.thenewboston.kotlinsdk.home.viewmodels
+
+import androidx.lifecycle.ViewModel
+import com.thenewboston.kotlinsdk.LIST_DATA_LIMIT_DEFAULT
+import com.thenewboston.kotlinsdk.home.repository.bank.BankRepo
+
+class BankViewModel(
+    private val repo: BankRepo
+) : ViewModel() {
```

Comment:
I recommend checking out this codelab on how to use LiveData. https://developer.android.com/codelabs/android-training-livedata-viewmodel#0

Reply:
Cool, will integrate livedata for the rest of the viewmodels as well.